addons/mail/static/src/core_ui/primary_variables.scss

```diff
@@ -15,6 +15,10 @@ $o-mail-Chatter-minWidth: 530px !default;
 $o-mail-Discuss-inspector: 250px !default;
 $o-mail-Discuss-headerHeight: 48px !default;
 
+.o-min-width-0 {
```

Comment:
it's a little bit weird to have a class like this because we don't know if this class will be generic enough to be put into `primary_variables.scss`, if so, `o-mw-0` can be a clean name already, coming from a similar class name from bootstrap

Reply:
`mw` from boostrap is used for `max-width`